### ISSUE TITLE
ModelDatacollection not working

### ISSUE THREAD

**Issue by Jerry1309** [Azure/MachineLearningNotebooks#1679]
Hi,
I'm trying to enable modeldatacollection by following the Microsoft documentation. however i'm getting the error 

"{\"error\": \"'ModelDataCollector' object has no attribute '_handle'\"}" scoring script is given below

    `import json
     import numpy as np
     import os
     import pickle
     from sklearn.externals import joblib
     from sklearn.linear_model import LogisticRegression
     from azureml.core.model import Model
     from azureml.monitoring import ModelDataCollector

    def init():
         global model

         model_filename = 'sal_model.pkl'
         model_path = os.path.join(os.environ['AZUREML_MODEL_DIR'], model_filename)

         model = joblib.load(model_path)
    
         global inputs_dc, prediction_dc

         inputs_dc = ModelDataCollector(model_name = "salarypredictor_model", designation="sal_inputs", feature_names=["feat1"])
         prediction_dc = ModelDataCollector(model_name = "salarypredictor_model", designation="sal_predictions", feature_names ["prediction1"])

     def run(data):
         try:
              data = np.array(json.loads(data)['data'])

                y_hat = model.predict(data)

            global inputs_dc, prediction_dc
            inputs_dc.collect(data, user_correlation_id='') 
            prediction_dc.collect(y_hat, user_correlation_id='') 

            return json.dumps(y_hat.tolist())

          except Exception as e:
           result = str(e)
           # return error message back to the client
           return json.dumps({"error": result})`

**Comment by Jerry1309** [comment]
*****************************issue resolved.*********************************

 
Incorrect designation value. Supported values: 'default', 'inputs', 'predictions', 'labels', 'signals', 'general'
and the feature name should be given in the format ["feature1","feature2"]

also pandas is a must dependency for the environment
